datajoint/table.py

```diff
@@ -1,4 +1,4 @@
-import collections
+import collections.abc
```

Comment:
Nm, this has actually been handled correctly already. We can leave as is.

```suggestion
import collections
```